docs/topics/mpp/multiplatform-library.md

```diff
@@ -240,33 +273,103 @@ class Base64JvmTest {
 }
 ```
 
-This test will be automatically executed on the JVM target in addition to the common part.
+This test will automatically run on the JVM platform in addition to the common tests.
 
-## Publishing library to Maven
+## Publish your library to the local Maven repository
 
-Our first multiplatform library is almost ready. The last step is to publish it, so other projects can then depend on our library.
-To make the publishing mechanism work, you should enable the experimental Gradle feature in `settings.gradle`:
+Your multiplatform library is ready for publishing so that you can use it in other projects.
 
-```groovy
-enableFeaturePreview('GRADLE_METADATA')
-```
+To publish your library, use the [`maven-publish` Gradle plugin](https://docs.gradle.org/current/userguide/publishing_maven.html).
+
+1. In the `build.gradle(.kts)` file, apply the `maven-publish` plugin and specify the group and version of your library:
 
-Now the classic `maven-publish` Gradle [plugin](https://docs.gradle.org/current/userguide/publishing_maven.html) can be used.
-Don't forget to specify the group and version of your library along with the plugin in `build.gradle`:
+<tabs>
 
 ```groovy
 apply plugin: 'maven-publish'
 group 'org.jetbrains.base64'
 version '1.0.0'
 ```
 
-Now check it with the command `./gradlew publishToMavenLocal` and you should see a successful build. 
-That's it, our library is now successfully published and any Kotlin project can depend on it, whether it is another common library, JVM, JS, or Native application.
+```kotlin
```

Comment:
Why does the Kotlin sample have `multiplatform` plugin and the Groovy one doesn't?

Reply:
I suggest adding multiplatform plugin to the Groovy example as well (though it's a section about publishing, you need both of the plugins to make it work) and update the plugins block [to the new style](https://stackoverflow.com/questions/32352816/what-the-difference-in-applying-gradle-plugin#:~:text=The%20plugins%20block%20is%20the,a%20plugin%20to%20your%20build.) so it would be consistent with Kotlin sample:

So the Groovy sample should look like this:
```

plugins {
    id 'org.jetbrains.kotlin.multiplatform' version '1.4.30'
    id 'maven-publish'
}

group = 'org.jetbrains.base64'
version = '1.0.0'

```

Reply:
Thanks @KaterinaPetrova and @p7nov ! updated.